website/docs/docs/cloud/manage-access/about-access.md

```diff
@@ -2,6 +2,8 @@
 title: "About user access in dbt Cloud"
 description: "Learn how dbt Cloud administrators can use dbt Cloud's permissioning model to control user-level access in a dbt Cloud account."
 id: "about-user-access"
+pagination_next: "docs/cloud/manage-access/audit-log"
```

Comment:
do we not want users to go to [seats and users](docs/cloud/manage-access/seats-and-users) first before audit log?

```suggestion
pagination_next: "docs/cloud/manage-access/audit-log"
```

Reply:
Good catch! Thank you